solidity/utils/addBNBs_mainnet.json

```diff
@@ -0,0 +1,34 @@
+{
+    "reserves": [
+        {
+            "symbol": "(WR)BTC",
+            "decimals": 18,
+            "address": "0x69FE5cEC81D5eF92600c1A0dB1F11986AB3758Ab"
```

Comment:
shouldn't this be 0x542fDA317318eBF1d3DEAf76E0b632741A7e677d ?

Reply:
Done 👍